draft-ietf-tls-esni.md

```diff
@@ -548,39 +562,61 @@ error code.
 
 ### HelloRetryRequest
 
-If the server sends a HelloRetryRequest in response to the ClientHello
-and the client can send a second updated ClientHello per the rules in
-{{RFC8446}}. At this point, the client does not know whether the
-server processed ClientHelloOuter or ClientHelloInner, and MUST
-regenerate both values to be acceptable. Note: if the inner and outer
-ClientHellos use different groups for their key shares or differ in
-some other way, then the HRR may actually be invalid for one or the
-other ClientHello. In that case, the Client MUST continue the
-handshake without changing the unaffected CH. Otherwise, the usual
-rules for HRR processing apply.
+If the server sends a HelloRetryRequest in response to the ClientHello,
+the client sends a second updated ClientHello per the rules in {{RFC8446}}.
+However, at this point, the client does not know whether the server processed
+ClientHelloOuter or ClientHelloInner, and MUST regenerate both values to
+be acceptable. Note: if the inner and outer ClientHellos use different groups
+for their key shares or differ in some other way, then the HRR may actually be
+invalid for one or the other ClientHello. In that case, the Client MUST continue
+the handshake without changing the unaffected CH. Otherwise, the usual rules for
```

Comment:
Nit: I would probably s/CH/ClientHello/ and s/HRR/HelloRetryRequest/ since they're not common abbreviations.